maintenance_project/views/maintenance_equipment_views.xml

```diff
@@ -40,10 +40,13 @@
                 <field
                     name="create_project_from_equipment"
                     attrs="{'invisible': [('id', '!=', False)]}"
+                    groups="project.group_project_manager"
                 />
+                <field name="project_id" invisible="1" />
```

Comment:
Not sure if this invisible field is needed. If it's because `project_id` is required by `preventive_default_task_id`, this field should be also restricted to `project.group_project_manager` as well.